reflex/components/disclosure/tabs.py

```diff
@@ -13,7 +18,7 @@ class Tabs(ChakraComponent):
     tag = "Tabs"
 
     # The alignment of the tabs ("center" | "end" | "start").
-    align: Var[str]
+    align: Var[LiteralTagAlign]
```

Comment:
```suggestion
    align: Var[LiteralTabsAlign]
```

This is just for the tabs yeah?